src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,9 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
```

Comment:
_In Main class create Machine array with Truck, Bulldozer and Excavator and call methods doWork() and stopWork() in a loop._